config/dwdirectiverules/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
```

Comment:
See  https://github.com/NearNodeFlash/nnf-sos/blob/master/config/dws/kustomization.yaml .  It shows you how nnf-sos places its own rules into the correct namespace so the DWS webhook can find it.

You'll need this after you change default/kustomization.yaml to go to the "user" namespace.